src/js/components/GroupNavDMButton/GroupNavDMButton.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from "react";
+import '../../../scss/components/GroupNavDMButton/GroupNavDMButton.scss';
+import PropTypes from "prop-types";
+import { GlobalContext } from "../../contexts/GlobalContextWrapper";
+
+const avatarClassNames = {
+  baseAvatar: 'avatar',
+  noAvatar: 'avatar-default',
+  availableAvatar: 'avatar-image',
+}
+
+const btnClasses = {
+  btnBase: 'nav-item-btn',
+  btnActive: 'nav-item-btn-active',
+}
+
+export const backgroundColorClassNames = {
+  avatarBlue: 'background-brand-blue',
+  avatarGreen: 'background-brand-green',
+  avatarOrange: 'background-brand-orange',
+  avatarPurple: 'background-brand-purple',
+  avatarRed: 'background-brand-red',
+  avatarYellow: 'background-brand-yellow',
+}
+
+class GroupNavDMButton extends Component {
+  static contextType = GlobalContext;
+  static defaultProps = {
+    title: 'Unnamed',
+    backgroundColor: backgroundColorClassNames.avatarBlue,
+  }
+
+  btnClasses = () => [
```

Comment:
I don't feel like being picky about it, but I think on a real-world project I'm gonna propose we DocBlock everything